companies.html

```diff
@@ -488,6 +488,17 @@
       Uses Scrapy to crawl and scrape taxi and transportation sites to gather rates, coverage zones, and deals.</p>
     </div>
 
+    <div class="company-box">
+      <a href="https://www.allogarage.fr/">
```

Comment:
https:// version does not load for me.

Also, is there any chance of you blogging or tweeting about how you use scrapy?
That would be great